lib/messages/flow/help.js

```diff
@@ -8,7 +8,15 @@ const commands = [
   },
   {
     usage: 'unsubscribe owner/repository',
-    desc: 'Unsubscribe to notifications for a repository',
+    desc: 'Unsubscribe from notifications for a repository',
+  },
+  {
+    usage: 'subscribe owner',
+    desc: 'Subscribe to notifications for an organization',
```

Comment:
Should we make explicit here that you "subscribe to notifications for all repositories in an organization"? Since there are a bunch of other org level events around teams, members, and so on that won't show up in Slack?